weave/wandb_api.py

```diff
@@ -332,6 +367,39 @@ def artifact_manifest_url(
             return None
         return file["directUrl"]
 
+    ARTIFACT_MANIFEST_FROM_ID_QUERY = gql.gql(
```

Comment:
duplicate?

Reply:
duplicate of? the other manifest query uses the entity, project and artifact name.. - or am i missing something? https://github.com/wandb/weave/blob/17febaa0e1fcc0c973295600d8e7370c95f046b0/weave/wandb_api.py#L288

Reply:
There are two copies of ARTIFACT_MANIFEST_FROM_ID_QUERY

Reply:
One's under `WandbApi` and the other under `WandbApiAsync`, like the other queries, or is it not supposed to be that way?

Reply:
oh i didn't realize that the original authors duplicated all these queries. Seems less than ideal to me (:

Reply:
probably should just be constants in the file

Reply:
i can update that in a follow up